.github/workflows/index.yaml

```diff
@@ -86,8 +87,8 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.RELEASE_TOKEN }}
         with:
-          tag_name: ${{ env.RELEASE_COMMIT_SHORT }}
-          release_name: ${{ env.RELEASE_COMMIT_SHORT }}
+          tag_name: ${{ env.RELEASE_NAME }}
+          release_name: ${{ env.RELEASE_NAME }}
```

Comment:
In clangds/clangd we include the `snapshot_` prefix in the tag name only, the release name is just for humans.

Maybe `index $RELEASE_DATE` or just `$RELEASE_DATE`?